app/views/govuk_component/option_select.raw.html.erb

```diff
@@ -0,0 +1,31 @@
+<div class="govuk-option-select" tabindex="0">
+  <div class="container-head js-container-head">
+    <div class='option-select-label'><%= title %></div>
+  </div>
+  <div class='options-container'>
+    <div class='js-auto-height-inner'>
+      <% options.each do | option |%>
+
+      <label for='<%= option['id'] %>'>
```

Comment:
These are safe - unless a string is explicitly marked as `html_safe` it'll get escaped automatically.